membership/src/main/kotlin/net/corda/v5/membership/GroupPolicyProperties.kt

```diff
@@ -0,0 +1,8 @@
+package net.corda.v5.membership
+
+import net.corda.v5.base.types.LayeredPropertyMap
+
+/**
+ * The properties of the group policy
+ */
+interface GroupPolicyProperties : LayeredPropertyMap
```

Comment:
This seems to only be used internally so could it be moved to the runtime-os? Or you think it needs to be public? We do have the `GroupPolicy` interface public already.